pkg/local_object_storage/blobstor/fstree/fstree.go

```diff
@@ -260,7 +281,12 @@ func (t *FSTree) PutStream(addr oid.Address, handler func(*os.File) error) error
 		return err
 	}
 
-	f, err := os.OpenFile(p, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, t.Permissions)
+	flags := os.O_WRONLY | os.O_CREATE | os.O_TRUNC
```

Comment:
Maybe also share `openFile` method? It can be used here and in `writeFile`.

Reply:
It is too small and can only be used for write IMO. Created `writeFlags` instead.